### DISCUSSION TITLE
Browser Reload Icon

### DISCUSSION THREAD

**Discussion by pondorasti** [tailwindlabs/heroicons#827] [Ideas]
I think a standard browser reload icon in addition to `arrow-path` would be a great addition to the collection. I've went ahead and made a modified version of `arrow-path` to see how it would look ([source](https://user-images.githubusercontent.com/32957606/191977040-2eb9e34b-cac6-4d17-be60-634c019f4522.svg)).

<img width="446" alt="CleanShot 2022-09-23 at 06 56 09@2x" src="https://user-images.githubusercontent.com/32957606/191977000-3b936f7d-6a7c-4f89-a521-c75e940aeb9a.png">

**Comment by ndri** [comment]
I added `arrow-path-single-clockwise` and `arrow-path-single-counterclockwise` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here:

* https://sidekickicons.com/?icon=arrow-path-single-clockwise
* https://sidekickicons.com/?icon=arrow-path-single-counterclockwise
